packages/react-core/src/components/Panel/examples/Panel.md

```diff
@@ -0,0 +1,171 @@
+---
+id: Panel
+section: components
+cssPrefix: pf-c-panel
+propComponents: [Panel, PanelMain, PanelMainBody, PanelHeader, PanelFooter]
+beta: true
+---
+
+## Examples
+
+### Basic
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody } from '@patternfly/react-core';
+
+const BasicPanel = () => (
+  <Panel>
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Header
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody, PanelHeader, Divider } from '@patternfly/react-core';
+
+const HeaderPanel = () => (
+  <Panel>
+    <PanelHeader>Header content</PanelHeader>
+    <Divider />
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Footer
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody, PanelFooter } from '@patternfly/react-core';
+
+const FooterPanel = () => (
+  <Panel>
+    <PanelMain>
+      <PanelMainBody>Main Content</PanelMainBody>
+    </PanelMain>
+    <PanelFooter>Footer content</PanelFooter>
+  </Panel>
+);
+```
+
+### Header and footer
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody, PanelHeader, Divider, PanelFooter } from '@patternfly/react-core';
+
+const HeaderFooterPanel = () => (
+  <Panel>
+    <PanelHeader>Header content</PanelHeader>
+    <Divider />
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+    <PanelFooter>Footer content</PanelFooter>
+  </Panel>
+);
+```
+
+### No body
+
+```js
+import React from 'react';
+import { Panel, PanelMain } from '@patternfly/react-core';
+
+const NoBodyPanel = () => (
+  <Panel>
+    <PanelMain>Main content</PanelMain>
+  </Panel>
+);
+```
+
+### Raised
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody } from '@patternfly/react-core';
+
+const RaisedPanel = () => (
+  <Panel isRaised>
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Bordered
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody } from '@patternfly/react-core';
+
+const BorderedPanel = () => (
+  <Panel isBordered>
+    <PanelMain>
+      <PanelMainBody>Main content</PanelMainBody>
+    </PanelMain>
+  </Panel>
+);
+```
+
+### Scrollable
+
+```js
+import React from 'react';
+import { Panel, PanelMain, PanelMainBody } from '@patternfly/react-core';
+
+const ScrollablePanel = () => {
+  const mainContent = (
+    <>
+      Main Content
+      <br />
+      <br />
```

Comment:
nit - can you remove these extra `br`s at the end of the content? Makes it look like there is extra space at the end of the body

how it looks now:
<img width="678" alt="Screen Shot 2021-11-10 at 2 37 42 PM" src="https://user-images.githubusercontent.com/35148959/141189765-3a262f77-dd5b-4b35-8ddf-bf4c1ab9ffe3.png">

without the trialing br's
<img width="529" alt="Screen Shot 2021-11-10 at 2 37 21 PM" src="https://user-images.githubusercontent.com/35148959/141189799-ce9371b9-2ae2-41c9-b40c-de30b59b9660.png">